libgloss/arc/emsdp1.1.ld

```diff
@@ -0,0 +1,18 @@
+/* Memory map for ARC EM Software Development Platform 1.0 and 1.1 */
+
+MEMORY
+{
+    PSRAM : ORIGIN = 0x10000000, LENGTH =  16M
+    ICCM0 : ORIGIN = 0x60000000, LENGTH = 128K
+    DCCM  : ORIGIN = 0x80000000, LENGTH = 128K
+}
+
+REGION_ALIAS("startup", ICCM0)
+REGION_ALIAS("text", ICCM0)
+REGION_ALIAS("data", DCCM)
+REGION_ALIAS("sdata", DCCM)
+
+PROVIDE (__stack_top = (0x8001ffff & -4));
```

Comment:
You don't need `__stack_top` and `__end_heap` as they are defined in the common part ;)

Reply:
Oops! Indeed.